### DISCUSSION TITLE
How to "Get Code" like in the Designer on the website?

### DISCUSSION THREAD

**Discussion by mparsakia** [pdfme/pdfme#106] [Q&A]
Hi, just started using this library and I imported the designer to my website (mostly so I could use custom fonts while designing) and would like to know, how can I "Get Code" for the design I have just created?

**Comment by hand-dot** [comment]
Hi @mparsakia .

You can use [getTemplate](https://pdfme.com/docs/api/ui/classes/Designer#gettemplate) method.
